hack/update-yamls-with-the-same-content.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+get_current_csi_version (){
+  current_csi_version=$(cat version/version.go | grep -i driverversion | awk -F = '{print $2}')
+  echo ${current_csi_version//\"}
+}
+
+are_csv_files_exsists_in_current_csi_version (){
+  current_csi_version=$(get_current_csi_version)
+  if ! compgen -G "${PWD}/deploy/olm-catalog/*/$current_csi_version" > /dev/null; then
+    exit 0
```

Comment:
the function name implies it is a boolean function, but this line shows that the function is not boolean.
please change this function to be boolean